src/plugins/csvstorage/README.md

```diff
@@ -0,0 +1,30 @@
+- infos = Information about the csvstorage plugin is in keys below
+- infos/author = Thomas Waser
```

Comment:
Can you provide some e-mail adr? You can also have a `@libelektra.org` forwarded to you.